src/rc_msg.c

```diff
@@ -209,13 +209,22 @@ int32_t
 rc_vprintf(uint32_t severity, const char *format, va_list ar)
 {
 	struct timeval tv;
+	#if LINUX_VERSION_CODE >= KERNEL_VERSION(5,0,0)
```

Comment:
Could you move all the if clauses for the compiler to the beginning of the line?
Just like it is done in src/rc_init.c?

Personally, I only saw people coding like this. But if you convince me of something different, then I'll agree on your code style of course :smile: